src/resources/js/main.ts

```diff
@@ -36,10 +61,10 @@ function jsApiFunction() {
             let json = await fetchPromise.json();
             hideLoader('dynamic-table');
 
-            json.results.map((person) => {
+            json.results.map((person: Person) => {
                 addPersonToTable(person);
             });
-        } catch (e) {
+        } catch (e: any) {
```

Comment:
errors are pretty annoying, I use `any` quite a bit for errors. Just if you wanted to try and type it, I'd say it's a good exercise

Reply:
`tsc` was throwing this for me when I changed it from `any`. I looked into it a little bit, and saw some different approaches to check the type of error and act accordingly. I'm definitely going to play around with this more once I get into API calls in the React pieces I'm working on. 
`TS1196: Catch clause variable type annotation must be ‘any’ or ‘unknown’ if specified.`

Reply:
whoa that's interesting, thanks for expanding, that's interesting. Great callout